roles/openshift_aws/defaults/main.yml

```diff
@@ -61,6 +61,48 @@ openshift_aws_vpc:
 #    - cidr: 172.31.16.0/20
 #      az: "us-east-1a"
 
+openshift_aws_create_dns: False
+openshift_aws_dns_provider: "route53"
+# openshift_aws_dns_zone: ""
+# ie. openshift_aws_dns_zone: "{{ openshift_aws_clusterid }}.example.com"
+
+openshift_aws_dns_records:
```

Comment:
How did this become a list again?

Reply:
because it's the only way to iterate it and extract attributes otherwise we lose the keys